examples/policies/kubernetes/clustermesh/cross-cluster-any-namespace-policy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "allow-cross-cluster-any-ns"
+  description: "Allow x-wing in cluster1 to contact rebel-base in cluster2 (in any NS)"
```

Comment:
The `description` field should be moved to the `spec` stanza, otherwise the policy is invalid. The other example in the same folder seems affected by this problems as well.

Reply:
Ugh, good catch! I just fixed that in the latest force commit. Thanks!

Reply:
Thanks!